docs/fidesops/docs/guides/configuration_reference.md

```diff
@@ -128,6 +126,25 @@ Please note: The configuration is case-sensitive, so the variables must be speci
 | `FIDESOPS__DATABASE__SQLALCHEMY_DATABASE_URI` | None | An optional override for the URI used for the database connection, in the form of `postgresql://<user>:<password>@<hostname>:<port>/<database>`. |
 | `TESTING` | False | This variable does not need to be set - Pytest will set it to True when running unit tests, so we run against the test database. |
 
+## Celery configuration
+
+Fidesops uses [Celery](https://docs.celeryq.dev/en/stable/index.html) for asynchronous task management. 
+
+The `celery.toml` file provided contains a brief configuration reference for managing Celery variables. By default, fidesops will look for this file in the root directory of your application, but this location can be optionally overridden by specifying an alternate `CELERY_CONFIG_PATH` in your `fidesops.toml`.
+
+For more information, see the [Celery configuration](https://docs.celeryq.dev/en/stable/userguide/configuration.html) documentation.
+
+```sh title="Example <code>celery.toml</code>"
+default_queue_name = "fidesops"
+broker_url = "redis://:testpassword@redis:6379/1"
+result_backend = "redis://:testpassword@redis:6379/1"
+```
```

Comment:
Would be good to weave in somewhere that there is a finite list of vars that can be overridden, specified here: https://docs.celeryq.dev/en/stable/userguide/configuration.html#new-lowercase-settings.